Декоративные элементы всех страниц

Maxgrbv opened the pull request:
---
:mortar_board: [Декоративные элементы всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2205519/tasks/13)

Maxgrbv committed:
Декоративные элементы всех страниц

senchkim commented on index.html:
```diff
@@ -11,44 +11,59 @@
-        </li>
-      </ul>
-</div>
+      <div class="navigation-company-container">
```
https://validator.w3.org/nu/#textarea
Есть ошибки в валидаторе

Maxgrbv replied:
> 

Добрый вечер! А как эти ошибки убрать? Кнопки с подсказками будут вырываться из потока, поэтому я их закомментировал, а как обращаться к svg иконкам без класса не совсем понятно

senchkim commented on styles/styles.css:
```diff
@@ -102,58 +112,92 @@ body {
-  padding: 8px;
+}
+
+/*  Не понимаю как сделать светлый background круглым, как на макете  */
```
`border-radius` - прочитай про него!
https://developer.mozilla.org/ru/docs/Web/CSS/border-radius

senchkim commented on styles/styles.css:
```diff
@@ -556,35 +642,35 @@ body {
-  color: #565C66;
-  width: 506px;
-  height: 66px;
+/*   Подскажите, пожалуйста, почему у формы и у кнопки в этом диве снизу padding, если он обнулен через теги по классам?   */
```
`delievery-form-container` - тут есть же маргин и он наследуется

senchkim commented on styles/styles.css:
```diff
@@ -826,9 +933,16 @@ body {
   max-width: 200px;
 }
 
+.copyright-list {
```
#Совет
Ты много раз дублируешь обнуление списка и дублируешь кучу кода!
Создай класс типа `.reset-list` (и пропиши в нем обнуление) - и задай его всем тэгам `ul` 👍

senchkim commented on styles/styles.css:
```diff
@@ -1007,50 +1137,347 @@ flex-direction: column;
+  list-style: none;
 }
 
+/*  Не понимаю почему цифры страниц в пагинации расположены слегка выше, чем должны быть  */
```
Не обращай внимания, все в пределах нормы 👍

senchkim commented on styles/styles.css:
```diff
@@ -1007,50 +1137,347 @@ flex-direction: column;
+}
+
+.button-white {
+  /*  Какое свойство имеет кнопка со спиннером?  */
```
Как понять какое свойство?
https://pastenow.ru/08c7f3f009eca72dd43e938954fe2f5b

senchkim commented on styles/styles.css:
```diff
@@ -1007,50 +1137,347 @@ flex-direction: column;
+.breadcrumbs-link {}
+
+.breadcrumbs-link-current {
+  border-bottom: 1px solid #2D3440;
```
И еще по поводу слайдера!
Верстаем и стилизуем все 3 слайда, но показываем только первый - остальные два просто скрываем 👍

Maxgrbv committed:
Fix. Декоративные элементы всех страниц

Добрый вечер! А как эти ошибки в валидаторе убрать? Кнопки с подсказками будут вырываться из потока, поэтому я их закомментировал, а как обращаться к svg иконкам без класса не совсем понятно.

> Как понять какое свойство?
https://pastenow.ru/08c7f3f009eca72dd43e938954fe2f5b

Имею в виду, что в стайлгайде есть кнопка с кружком загрузки, называется "кнопка со спиннером"
есть button:hover, button:active и т.д, а у этой какое должно быть значение, если не hover?

Спасибо)

Maxgrbv committed:
Update catalog.html
